src/util/to-markdown.ts

```diff
@@ -28,18 +30,49 @@ export function isMarkdown(html: string): boolean {
 
 export async function toMarkdown(html: string): Promise<MarkdownResult> {
   let markdown = escape(turndown.turndown(html.trim()).trim());
+  let addjsReplacements = 0;
   if (hasAddjsLink(markdown)) {
+    markdown = markdown.replace('addjs="', 'addjs src="'); //Replace the one occurrence of missing ` src`
     const matches =
-      markdown.match(/(\\)?\[addjs\s*src="[^"\\]+(["\\\]]+)?/g) || [];
+      // TODO: This regex handles everything but the missing "src" problem above
+      // Handles " and ', missing closing quotes, no-break space chars
+      // \[(addjs)[\p{Z}]?src=["']([^"'\]]+)["'\]]+
+      //markdown.match(/\[(addjs)[\p{Z}]+src=["']([^"'\\\]]+)["'\\\]]+/g) || [];
+      //markdown.match(/(\\)?\[addjs\s+src="[^"\\]+(["\\\]]+)?/g) || [];
+      //markdown.match(/\[(addjs)[\s|\u00A0]+src=["']([^"'\\\]]+)["'\\\]]+/g) || [];
+      // For some reason \p{Z} isn't matching, so use \u00A0 instead
+      markdown.match(/\[(addjs)[\s\u00A0]+src=["']([^"'\\\]]+)["'\\\]]+/g) ||
+      [];
+    //
+    let expecting = 0;
+    let expectedMatches = '';
+
+    for (let match of markdown.match(/addjs.{50}/g) || []) {
+      expecting++;
+      expectedMatches = expectedMatches + match + '\n';
+    }
+
+    let i = 0;
+
     for (let match of matches) {
```

Comment:
This may as well just be a traditional for loop if you're gonna keep track of an index :)

Reply:
I think I was planning on taking this code out once I had all of the matches accounted for.  I'm still missing like 5 for some reason.